app/controllers/api/account.php

```diff
@@ -2481,13 +2469,12 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
             }
         }
 
-        // Set to unhashed secret for events and server responses
         $token->setAttribute('secret', $secret);
 
         $queueForEvents
             ->setPayload($response->output($token, Response::MODEL_TOKEN), sensitive: ['secret']);
 
-        // Hide secret for clients
+        // Encode secret for clients
         $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? Auth::encodeSession($user->getId(), $secret) : '');
```

Comment:
Can't we remove the hide logic now?
```suggestion
        $token->setAttribute('secret', Auth::encodeSession($user->getId(), $secret));
```